internal/proxy/task_scheduler.go

```diff
@@ -170,14 +171,24 @@ func (queue *baseTaskQueue) Enqueue(t task) error {
 		return err
 	}
 
-	ts, err := queue.tsoAllocatorIns.AllocOne(t.TraceCtx())
-	if err != nil {
-		return err
+	var ts Timestamp
+	var id UniqueID
+	if t.CanSkipAllocTimestamp() {
```

Comment:
can we skip alloc timestamp for other operations?

Reply:
No, for CreateCollection,  cannot skip alloc timestamp.